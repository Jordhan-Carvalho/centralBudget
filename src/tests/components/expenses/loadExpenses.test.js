import React from 'react';
import { shallow } from 'enzyme';
import LoadExpenses from '../../../components/expenses/loadExpenses';

test('should render expenses page', () => {
    const wrapper = shallow(<LoadExpenses />);
    expect(wrapper).toMatchSnapshot();
});