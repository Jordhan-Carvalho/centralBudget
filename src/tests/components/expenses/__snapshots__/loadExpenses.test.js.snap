// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expenses page 1`] = `
<Fragment>
  <h1>
    Expense List
  </h1>
  <hr />
  <ConnectFunction />
  <ConnectFunction
    type="expense"
  />
  <hr />
  <ConnectFunction />
  <hr />
  <Link
    to="/create"
  >
    <button
      className="btn btn-dark"
    >
      Add Expense
    </button>
  </Link>
</Fragment>
`;
